audio/data.py

```diff
@@ -365,11 +368,18 @@ def create_spectro(self, item:AudioItem):
         if self.config.mfcc: 
             mel = MFCC(sample_rate=item.sr, n_mfcc=self.config.sg_cfg.n_mfcc, melkwargs=self.config.sg_cfg.mel_args())(item.sig)
         else:
-            if self.config.sg_cfg.n_mels > 0:
-              mel = MelSpectrogram(**(self.config.sg_cfg.mel_args()))(item.sig)
+            if self.config.sg_cfg.custom_spectro != None:
+                mel = self.config.sg_cfg.custom_spectro(item.sig)
             else:
-              mel = Spectrogram(**(self.config.sg_cfg.spectro_args()))(item.sig)
-            
+                if self.config.sg_cfg.n_mels > 0:
```

Comment:
For the record: I generally dislike using magic values for numeric parameters as flags. Since this is a temporary adaptation of an already hacky library, I don't mind, though.